core/src/main/resources/jenkins/agents/CloudSet/index.jelly

```diff
@@ -27,7 +27,8 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
-  <l:layout title="${it.displayName}" type="one-column">
```

Comment:
-1 page looks unbalanced, and off. see review comments in https://github.com/jenkinsci/cloud-stats-plugin/pull/73